quilt3_local/README.md

```diff
@@ -0,0 +1,68 @@
+# Quilt Lite
+
+Open source implementation of the Quilt registry that works in the local
+environment (not requiring AWS cloud services).
+
+## TL;DR
+
+```shell
+# set up venv, assuming poetry is available in the PATH
+poetry install
+
+# set up and build catalog
+(cd ../catalog && npm i && npm run build)
+
+# run the app at http://localhost:3000
+poetry run uvicorn quilt3_local.main:app --reload --port 3000
+```
+
+## Set-up
+
+### Python environment set-up
+
+First, you need [`poetry` installed](https://python-poetry.org/docs/#installation).
+
+Then, you have to set up the virtualenv by running `poetry install` from the
+project directory -- it *should* install all the requirements
+(consider removing `../api/python/quilt3.egg-info/` directory if dependndency
+resolution doesn't work or takes too long -- it contains cached dependency list
+(and other metadata) of the `quilt3` package which is being installed as a local
+dependency).
+
+### Catalog (node.js) environment set-up
+
+Catalog supports `node` version `16` and `npm` versions `7` and `8`.
+
+To install the dependencies simply run `npm i` from the catalog root directory
+(`../catalog/`).
```

Comment:
I feel like this should be in README for catalog and we should have just a reference here.

Reply:
yeah it makes sense